source/atomics.tex

```diff
@@ -791,14 +791,15 @@
 
 \pnum
 \begin{note}
-Hardware could require an object
+Hardware can require an object
 referenced by an \tcode{atomic_ref}
 to have stricter alignment\iref{basic.align}
 than other objects of type \tcode{T}.
 Further, whether operations on an \tcode{atomic_ref}
-are lock-free could depend on the alignment of the referenced object.
-For example, lock-free operations on \tcode{std::complex<double>}
-could be supported only if aligned to \tcode{2*alignof(double)}.
+are lock-free can depend on the alignment of the referenced object.
+For example, it is possible for
+lock-free operations on \tcode{std::complex<double>}
+to be supported only if aligned to \tcode{2*alignof(double)}.
```

Comment:
Maybe "For example, operations referencing a `std::complex<double>` object that is not aligned to `2*alignof(double)`" are not guaranteed to be lock-free."

Reply:
That's a change in meaning, because it loses the twist "they are indeed lock-free if properly aligned".

Reply:
I've rephrased slightly + force-pushed. Maybe you like this better.